tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/repository/PostFundingRepository.java

```diff
@@ -11,10 +11,9 @@
 @SuppressWarnings("unused")
 public interface PostFundingRepository extends JpaRepository<PostFunding, Long> {
 
-  @Query("SELECT count(id) FROM PostFunding\n"
-      + "WHERE postId = :postId\n"
-      + "AND startDate IS NOT NULL\n"
-      + "AND (endDate >= current_date() OR endDate IS NULL)")
+  @Query("SELECT count(id) FROM PostFunding "
+      + "WHERE postId = :postId "
+      + "AND (endDate >= current_date() OR"
+      + "(endDate IS NULL AND startDate IS NOT NULL))")
```

Comment:
There're post fundings with null startDate, so for the post funding with null startDate and endDate >= current_date, I think we still need to count it as CURRENT.